Post page header updates

s-cheng opened the pull request:
This PR is part of a broader project to update the design of the post page header on the EA Forum. Most of the changes should only affect the EA Forum, but one piece I kept in for LW since I thought it was an overall improvement. :) [Project figma here.](https://www.figma.com/file/d09iVzNSpZp9GyWGcfkWmX/Q2.2-2023?type=design&node-id=13-10576&t=N9AcuSwkuoGxzq4J-0)

In this PR, I've just done the bits that reorganize existing parts of the post page header. The major things are:
1. On wide screens, karma/voting has moved to the left of the title, and the title & secondary info take up the full col width. This makes the layout more consistent with post items.
2. On wide screens, the secondary info is split into two sections, which makes it easier to parse.
3. If the secondary info row would wrap, we split it at the end of the authors list, so the rest of that info is on a separate line. (<- **this is the only one that I didn't forum-gate**)
4. There's now a border between the post header and the tags, to help make the right-side icons not feel like they are floating.

<img width="931" alt="Screen Shot 2023-06-15 at 1 58 23 AM" src="https://github.com/ForumMagnum/ForumMagnum/assets/9057804/5731c890-685b-45aa-872e-4e6577e0edee">
<img width="363" alt="Screen Shot 2023-06-15 at 2 01 49 AM" src="https://github.com/ForumMagnum/ForumMagnum/assets/9057804/5213ddfd-7413-4acf-9714-357073bec0b7">
<img width="1208" alt="Screen Shot 2023-06-15 at 2 16 01 PM" src="https://github.com/ForumMagnum/ForumMagnum/assets/9057804/fdefa04e-c759-42e9-b1fa-a9b8c0cf7b67">

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1204837482038052) by [Unito](https://www.unito.io)

s-cheng commented on packages/lesswrong/components/posts/PostsPage/PostsPageDate.tsx:
```diff
@@ -50,13 +50,11 @@ const PostsPageDate = ({ post, hasMajorRevision, classes }: {
-        </span>
-    </LWTooltip>
-  </React.Fragment>);
+  return <LWTooltip title={tooltip} placement="bottom">
+    <span className={classes.date}>
+      <FormatDate date={post.postedAt} format={format} tooltip={false} />
+    </span>
+  </LWTooltip>
```
This is just code cleanup, no functionality change

darkruby501 commented:
I like this change! Thanks

jpaddison3 commented on packages/lesswrong/components/posts/PostsPage/PostsPagePostHeader.tsx:
```diff
@@ -271,6 +293,57 @@ const PostsPagePostHeader = ({post, answers = [], dialogueResponses = [], toggle
+    </span>
+  
+  // this is the info section under the post title, to the right of the author names
+  let secondaryInfoNode = <div className={classes.secondaryInfo}>
```
This section of this file is really pleasant to read. Nice work

jpaddison3 reviewed (approved):
LGTM

s-cheng commented:
@agnesstenlund Thanks so much! ^^ I've fixed the action icon color, and I also moved the post page voting tooltips to the left of the arrows so it doesn't overlap the post title.
